### PR TITLE
AP-8080 - fixing checkout and offline orders.

### PR DESCRIPTION
**Pull request by Pro777**

### CODE REVIEW

File: app/code/Apruve/Payment/Model/PaymentMethod.php [hunk 1]
```diff
@@ -7,8 +7,10 @@ class PaymentMethod extends \Magento\Payment\Model\Method\AbstractMethod
     const CODE = 'apruve';
     const DISCOUNT = 'Discount';
 
+    const AUTHORIZE_ACTION = 'finalize';
+    const OFFLINE_ACTION = '';
```
Comment (simonbtomlinson) [inline app/code/Apruve/Payment/Model/PaymentMethod.php]: Why did this become blank?

File: app/code/Apruve/Payment/Observer/ShipmentObserver.php [hunk 2]
```diff
@@ -242,56 +244,6 @@ protected function _getDiscountItem($object)
         }
     }
 
-    protected function _apruve($action, $token, $data = '', $object = 'orders', $requestType = 'POST')
```
Comment (simonbtomlinson) [inline app/code/Apruve/Payment/Observer/ShipmentObserver.php]: Why was this deleted? I though this was how we did network calls?

Comment (Pro777) [reply app/code/Apruve/Payment/Observer/ShipmentObserver.php]: We use _processShipment now, because this call requires two completely different Nouns than _apruve was built to use.

File: app/code/Apruve/Payment/Model/PaymentMethod.php (hunk 1 above)
Comment (Pro777) [reply app/code/Apruve/Payment/Model/PaymentMethod.php]: I made that OFFLINE_ACTION blank because that is what is required to make the _apruve method call work for offline orders. I just wanted to make this more explicit.
